cpp/harvest/nacjd.cc

```diff
@@ -343,9 +355,14 @@ void ParseJSONAndWriteMARC(MARC::Writer * const title_writer) {
 
 
 int Main(int argc, char *argv[]) {
-    if (argc != 3)
+    if (argc < 3 or argc > 4)
         Usage();
 
+    int count_rows(9000);
```

Comment:
count_rows should always have a positive value, so instead of `int` you should use `unsigned` (Also in the function headers).